scripts/brp-compress

```diff
@@ -11,12 +11,8 @@ cd "$RPM_BUILD_ROOT"
 COMPRESS=${COMPRESS:-gzip -9 -n}
 COMPRESS_EXT=${COMPRESS_EXT:-.gz}
 
-for d in ./usr/man/man* ./usr/man/*/man* ./usr/info \
-	./usr/share/man/man* ./usr/share/man/*/man* ./usr/share/info \
-	./usr/kerberos/man ./usr/X11R6/man/man* ./usr/lib/perl5/man/man* \
-	./usr/share/doc/*/man/man* ./usr/lib/*/man/man* ./usr/share/fish/man/man*
+for d in `find . -type d -path "*/man/man*" -o -path "*/share/info"`
```

Comment:
this doesn't cover `/usr/info`, `/usr/kerberos/man` cases... I'm not saying that we should support it, but at least we should consider it.

Reply:
I am less concerned about /usr/info and /usr/kerberos/man because these seem to be some long forgotten legacy. 
Wondering more about */man/*/man*